src/prebid.js

```diff
@@ -90,6 +90,16 @@ function init(timeout, adUnitCodeArr) {
 	}
 	//set timeout for all bids
 	setTimeout(bidmanager.executeCallback, cbTimeout);
+
+	//set size by dynamic size
+	for(var j=0; j<pbjs.adUnits.length; j++){
```

Comment:
What is the external API? How can a publisher use this feature? We need example code please.